docs/guide/personalization/enabling_personalization.md

```diff
@@ -0,0 +1,54 @@
+# Enabling Personalization
+
+The Ibexa Personalization solution is based on a client-server architecture.
+To make it operational, you must obtain and set up certain authentication parameters.
+
+## Requesting access to the server
+
+At first you must request credentials required to access the recommendation server.
+
+To do this, in the Back Office, select **Personalization** and then **Dashboard**.
+On the welcome screen, enter the following details in their respective fields:
+
+* a full name of the person responsible for accepting the terms and conditions of the Personalization service
+* an e-mail address to which you want the credentials to be sent to
+* an Installation key that can be found on the **Maintenance and Support agreement details** page in the service portal
+
+Select a checkbox to confirm that you've read the terms and conditions, and then click **Submit**.
+Your request is sent to Ibexa, and you will receive a confirmation e-mail in response.
+
+## Configuring mandator credentials
```

Comment:
Idea for future - is the term "mandator" used in the UI? It's counter-intuitive, so maybe we could replace it with a different term in customer-facing texts?

Reply:
I'd personally go with "user". However, I've checked that there are occurrences in the doc, and Damian seems to like using it.

Reply:
it is not about liking but rather consistency